Runtime/CesiumFlyToController.cs

```diff
@@ -222,62 +237,67 @@ private void HandleFlightStep(float deltaTime)
                 return;
             }
 
-            this._currentFlyToTime += (double)deltaTime;
+            this._currentFlyToTime += deltaTime;
 
-            // If we reached the end, set actual destination location and orientation
+            double flyPercentage = 0.0;
             if (this._currentFlyToTime >= this._flyToDuration)
+            {
+                flyPercentage = 1.0;
+            }
+            else if (this._flyToProgressCurve != null && this._flyToProgressCurve.length > 0)
```

Comment:
I like the length check here. This doesn't exist on the Unreal version, which might be a nice improvement.